package.json

```diff
@@ -104,6 +104,7 @@
   "devDependencies": {
     "@kadira/storybook": "^2.35.3",
     "babel-eslint": "^7.2.2",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.5",
```

Comment:
move to `depedencies`. `rails assets: precompile` runs `yarn install` with `NODE_ENV=production`.